lms/templates/student_account/finish_auth.html

```diff
@@ -9,32 +9,9 @@
 </%block>
 
 <%block name="headextra">
-
```

Comment:
I would prefer that ``isExternal`` be declared as a formal RequireJS dependency. A quick analysis shows that there is a single usage in Studio that does access it through RequireJS. There are then two usages in LMS, this one and then one in AccessView that unfortunately is a non-RequireJS file. 

My recommendation is to treat ``isExternal`` as a shimmed library like JQuery, so it will be declared as 'empty:' for the sake of Optimizer. At least then the dependency is made explicit, and it will be easier to refactor in the future.